ls1intum/Artemis
# New instructor button to remove all students from exam

### Checklist

### Motivation and Context


Currently when instructors want to remove all students from an exam they have to remove them all by hand. In this PR we fix this.

### Description

Solve https://github.com/ls1intum/Artemis/issues/2935, by introducing a new button to remove all students at once. 

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration -> Exams -> students
3. test the two options: removing all students from the exam with the check box checked will delete all participations and submissions as well. 
4. Without the checkout, the participations and submissions should still exist, and all students only go unregistered from the exam without deleting existing stuff. 

### Test Coverage



Client:
exam-students.component: 83.33%

Server:
ExamResource: 95%
ExamRegistrationService: 94%

### Screenshots


<img width="722" alt="Bildschirmfoto 2021-03-13 um 00 41 17" src="https://user-images.githubusercontent.com/42205689/111009819-f2647380-8394-11eb-90bd-5d594c2ab220.png">

![Bildschirmfoto 2021-03-08 um 17 35 43](https://user-images.githubusercontent.com/42205689/110351106-c2d00700-8034-11eb-9053-99028226d580.png)
